examples/db-connect.php

```diff
@@ -1,37 +1,46 @@
 <?php
+declare( strict_types = 1 );
 
-$dotenv = Dotenv\Dotenv::createImmutable(dirname(__DIR__));
+use Dotenv\Dotenv;
+
+$dotenv = Dotenv\Dotenv::createImmutable( dirname( __DIR__ ) );
 $dotenv->load();
 
 use Illuminate\Database\Capsule\Manager as Capsule;
 
 $capsule = new Capsule;
-$capsule->addConnection([
-    'driver'    => 'mysql',
-    'host'      => 'localhost',
-    'database'  => getenv('DATABASE'),
-    'username'  => getenv('USERNAME'),
-    'password'  => getenv('PASSWORD'),
-    'charset'   => 'utf8',
-    'collation' => 'utf8_unicode_ci',
-    'prefix'    => '',
-]);
+$capsule->addConnection(
+	[
+		'driver' => 'mysql',
+		'host' => 'localhost',
+		'database' => getenv( 'DATABASE' ),
+		'username' => getenv( 'USERNAME' ),
+		'password' => getenv( 'PASSWORD' ),
+		'charset' => 'utf8',
+		'collation' => 'utf8_unicode_ci',
+		'prefix' => '',
+	]
+);
 
 // Make this Capsule instance available globally via static methods... (optional)
 $capsule->setAsGlobal();
 
-function savePush(APNSPush $push, $token) {
+// phpcs:ignore NeutronStandard.Globals.DisallowGlobalFunctions.GlobalFunctions,WordPress.NamingConventions.ValidFunctionName.FunctionNameInvalid
+function savePush( APNSPush $push, $token ) {
 	global $capsule;
 
-	$capsule->table('mobile_push_queue')->insert([
-		'token'					=> $token,
-		'payload'				=> json_encode($push),
-		'when'					=> date("Y-m-d H:i:s"),
-		'mobile_push_client_id'	=> 0,
-	]);
+	$capsule->table( 'mobile_push_queue' )->insert(
+		[
+			'token' => $token,
+			'payload' => json_encode( $push ),
+			'when' => gmdate( 'Y-m-d H:i:s' ),
```

Comment:
> date() is affected by runtime timezone changes which can cause date/time to be incorrectly displayed. Use gmdate() instead.

I guess this is safe in the context of this example? I checked [the docs](https://www.php.net/manual/en/function.gmdate.php) and I can see the value of forcing the time  formatting to be GMT/UTC.

On the other hand, the example script there prints the same string for me for both `date` and `gmdate`.

```
// This was run in Melbourne, Australia, GMT+10
$ php -r 'echo date("M d Y H:i:s", mktime(0, 0, 0, 1, 1, 1998));'                                                                                                         
Jan 01 1998 00:00:00%

$ php -r 'echo gmdate("M d Y H:i:s", mktime(0, 0, 0, 1, 1, 1998));'                                                                                                         
Jan 01 1998 00:00:00%
```

:shrug:

Reply:
Yep this makes total sense here – as long as we're using it consistently, it really doesn't matter which we use.